packages/grafana-runtime/src/utils/DataSourceWithBackend.ts

```diff
@@ -202,10 +202,23 @@ class DataSourceWithBackend<
     headers[PluginRequestHeaders.PluginID] = Array.from(pluginIDs).join(', ');
     headers[PluginRequestHeaders.DatasourceUID] = Array.from(dsUIDs).join(', ');
 
+    let queryStrObj: Record<string, string> = {};
+
     let url = '/api/ds/query';
```

Comment:
```suggestion
    let url = '/api/ds/query?type='+this.id;
```
then the rest can be:
```
url += '&key=var'
```

and type is the one i am most curious to see in the network tab!

Reply:
`type` is a bit ambiguous here. is it query type or ds type? maybe dstype?

doesnt matter if query is always same type, but could be instant vs range?

Reply:
now `ds_type`